docs/cse/rules/before-writing-custom-rule.md

```diff
@@ -107,3 +107,14 @@ You can use an expression like this example in any rule type. Here is an examp
 
 <img src={useBaseUrl('img/cse/example-in-editor.png')} alt="Example in editor" width="500"/>
 
+## Degraded rules
+
+A degraded rule is one that has had a portion of the rule shut off because it causes the rule to exceed a processing limit. If you write a custom rule that becomes degraded, you must tune the rule to correct the problem.
```

Comment:
```suggestion
A degraded rule is one that has had a portion of the rule shut off to prevent it from exceeding a processing limit. If you write a custom rule that becomes degraded, you must tune the rule to correct the problem.
```